doc/api/timers.md

```diff
@@ -74,6 +74,21 @@ to call `timeout.ref()` unless `timeout.unref()` had been called previously.
 
 Returns a reference to the `Timeout`.
 
+### timeout.refresh()
+<!-- YAML
+added: REPLACEME
+-->
+
+Sets the timer's start time to the current time, and reschedules the timer to
+call its callback at the previously specified duration adjusted to the current
+time. This is useful for refreshing a timer without allocating a new
+JavaScript object.
+
+Using this on a timer that has already called its callback will reactivate the
+timer.
+
+Returns a reference to the `Timeout`.
```

Comment:
```
* Returns: {Timeout}
```

Reply:
Can be unified with changes in 65d97c96aa59517f90c34bf5a6b962e1b5e42316